modules/ROOT/pages/managing-users.adoc

```diff
@@ -90,6 +90,10 @@ If enabled, the SAML assertions sent from the IdP must be encrypted and follow t
 +
 . Sign out of Anypoint Platform, navigate to the sign-on URL you entered in the *Identity Management SAML 2.0* form, then sign in through your identity provider to test the configuration.
 
+== Export SAML Metadata
+
+If your identity provider supports uploading service provider metadata, you can click the *Anypoint service provider metadata* link to download an XML file of your SAML configuration settings. You can then upload this file to your identity provider to configure Anypoint Platform.
```

Comment:
"service provider metadata": when two nouns are used as a single adjective to modify a third noun, place a hyphen between those two nouns. In this case, "service" and "provider" are being used as a single adjective to modify "metadata". So, "service provider metadata" should be "service-provider metadata". Otherwise, you get a noun string, and those can be difficult to make sense of.